plugins/lighthouse/src/components/AuditList/AuditListTable.tsx

```diff
@@ -66,34 +45,13 @@ const useStyles = makeStyles(theme => ({
   sparklinesCell: { minWidth: 120 },
 }));
 
-type SparklinesDataByCategory = Record<LighthouseCategoryId, number[]>;
-function buildSparklinesDataForItem(item: Website): SparklinesDataByCategory {
-  return item.audits
-    .filter(
-      (audit: Audit): audit is AuditCompleted => audit.status === 'COMPLETED',
-    )
-    .reduce((scores, audit) => {
-      Object.values(audit.categories).forEach(category => {
-        scores[category.id] = scores[category.id] || [];
-        scores[category.id].unshift(category.score);
-      });
-
-      // edge case: if only one audit exists, force a "flat" sparkline
-      Object.values(scores).forEach(arr => {
-        if (arr.length === 1) arr.push(arr[0]);
-      });
-
-      return scores;
-    }, {} as SparklinesDataByCategory);
-}
-
 export const AuditListTable: FC<{ items: Website[] }> = ({ items }) => {
   const classes = useStyles();
   const categorySparklines: Record<string, SparklinesDataByCategory> = useMemo(
     () =>
       items.reduce(
         (res, item) => ({
-          ...res,
+           ...res,
```

Comment:
accidental change; set editor to prettier on save

Reply:
This was actually an accident lol